# HG changeset patch
# User Michael Godfrey <michaeldgodfrey@gmail.com>
# Date 1380601366 14400
#      Tue Oct 01 00:22:46 2013 -0400
# Node ID 797545fa86d0ad69b7df78d8ca812d79ff2540b2
# Parent  968c99137122e98f68080e6c52ab8a4a7bc77dc6
plot.txi: fix \deg in TeX symbol table.

* doc/interpreter/plot.txi: fix \deg in TeX symbol table.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -863,17 +863,17 @@ width 0.6pt \tabskip=0pt\cr
 {& \hbox to \cola{$\backslash$neq}   && $\neq$
 && \hbox to \colb{}   && 
 && \hbox to \colc{}   &&   &\cr}
 \vskip12pt
 \symtable{Other (not in Appendix F Tables)} {#}
 {& \hbox to \cola{$\backslash$ldots}     && $\ldots$
 && \hbox to \colb{$\backslash$0}         && $\oslash$
 && \hbox to \colc{$\backslash$copyright} && $\copyright$      &\cr
-& $\backslash$deg        && $\deg$
+& $\backslash$deg        && $^\circ$
 &&    &&  
 &&    &&       &\cr}
 \vskip12pt
 \hskip36pt Table 15.1: Available special characters in \TeX\ mode (cont.)
 @end tex
 @end float
 
 A complete example showing the capabilities of the extended text is
